flake.nix

```diff
@@ -88,7 +88,24 @@
           });
     in
     {
-      packages.default = scope.dune;
+      packages = {
+        dune = scope.dune;
+        default = with pkgs; stdenv.mkDerivation rec {
+          pname = package;
+          version = "n/a";
+          src = ./.;
+          nativeBuildInputs = with ocamlPackages; [ ocaml findlib ];
```

Comment:
Why do we need findlib? We don't use any OCaml libraries. Or is it for the installation step?

Reply:
Yup, findlib has a `setupHook` ([source](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/ocaml/findlib/default.nix#L30)) that sets the `OCAMLFIND_DESTDIR` variable.

Reply:
Btw if you ever need to use opam inside this flake, be aware that `OCAMLFIND_DESTDIR` will break package builds in opam so make sure to unset it.